test/parallel_api/dynamic_selection/sycl/test_fixed_resource_policy_sycl.pass.cpp

```diff
@@ -7,56 +7,64 @@
 //
 //===----------------------------------------------------------------------===//
 
-#include <iostream>
 #include "oneapi/dpl/dynamic_selection"
+#include <iostream>
 #include "support/test_dynamic_selection_utils.h"
 #include "support/test_config.h"
+#include "support/utils.h"
 #if TEST_DYNAMIC_SELECTION_AVAILABLE
 #    include "support/sycl_sanity.h"
-#endif
+#endif // TEST_DYNAMIC_SELECTION_AVAILABLE
 
 int
 main()
 {
+    bool bProcessed = false;
+
 #if TEST_DYNAMIC_SELECTION_AVAILABLE
     using policy_t = oneapi::dpl::experimental::fixed_resource_policy<oneapi::dpl::experimental::sycl_backend>;
     std::vector<sycl::queue> u;
     build_universe(u);
     std::cout << "UNIVERSE SIZE " << u.size() << std::endl;
```

Comment:
I believe we should avoid output into std::cout
Or if required we may do it under some define...
Please fix here and in all other places.

Reply:
This is actually just informational. I don't know if there are informational macros currently in oneDPL that can be used. I'll remove this one.